spec/requests/api/moves_controller_create_v2_spec.rb

```diff
@@ -513,6 +513,32 @@
         end
       end
     end
+
+    context 'when the profile is for a Cat A prisoner' do
+      let(:profile) { create(:profile, :category_a) }
+      let(:errors_422) do
+        [
+          {
+            'title' => 'Unprocessable entity',
+            'detail' => "Profile person is a category 'a' prisoner and cannot be moved using this service",
```

Comment:
Super minor, but should this be capital `A`?

Reply:
very good question, in our API controller we `.humanize` the detail message, which in turn downcases the `A`:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/controllers/api_controller.rb#L182

Reply:
Ah, that's a bit inconvenient :D We've had to end up implementing words that are exceptions in other projects, so that things like HMP and YOI don't get downcased.

Reply:
yep its some annoying rails magic :-|